config/stage.json

```diff
@@ -65,6 +65,7 @@
 		"layout/query-selected-editor": true,
 		"legal-updates-banner": false,
 		"login/magic-login": true,
+		"login/qr-code-login": false,
```

Comment:
I think it would work ok to just omit the value from these configs

Reply:
I second this.

Reply:
Good call.